### DISCUSSION TITLE
Usage of variables in Qfield app

### DISCUSSION THREAD

**Discussion by MonikaShah** [opengisch/QField#3207] [Q&A]
I would like to know the usage of variables present in Qfield app. 
For example if I create a new variable called "qfield_username" and add "Monika Shah" to it. 
How can I use this variable in 
1. my postgis layer
2. gpkg layer.
Basically I need that if once the user inputs her name in the vairables of  qfield app, this should automatically get stored as one of the attributes in the layer. 
Is it possible ?

**Comment by suricactus** [comment]
Yes, it is, `cloud_username` and `cloud_useremail`.
